packages/client/lib/Chain.js

```diff
@@ -215,11 +200,29 @@ export default class Chain {
    * @param {!string} to - Recepient address.
    * @param {!string} value - Value of transaction.
    * @param {!string} data - Data to be passed to the transaction.
-   * @param {!string} from - The address from which the message is signed.
+   * @param {!string} [fee] - Fee price in native unit (e.g. sat/b, wei)
```

Comment:
I think we need `from` address for ETH.

Reply:
I checked to see if we have any uses for the from field and i couldn't find any. That's why i opted to replace the parameter so it's easier to use the fee (you don't always have to pass undefined for `from`)
Actually since this is in the abstraction layer interface, "from" is not really valid other than for ethereum
maybe you are thinking of multiple account support. that one actually needs thought how to add but in this interface, it would introduce ethereum conditional code into the apps.